decentraland/unity-explorer
# Fix: add mouse sensitivity settings in menu

## What does this PR change?


Fix #2196 

Adds two sliders in the "Control" portion of the settings menu in order to calibrate the horizontal/vertical mouse sensitivity

## How to test the changes?



1. Launch the explorer
2. Open the explore panel
3. Go to Settings-> Controls and modify the slider values
4. Check whether it affects the camera movements sensitivity
5. Logout
6. Re-launch the explorer
7. Check whether the mouse sensitivity values are loaded according to the previous run and they are applied to the camera movement

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
